### DISCUSSION TITLE
Are there any android applications that uses maplibre as a full fledged maps (google maps alternative)

### DISCUSSION THREAD

**Discussion by Raj2032** [maplibre/maplibre-rs#178] [General]
Hi, I was wondering if there were any apps that utilises maplibre as a full fledged working maps application which can be used as an alternative to google maps?

**Comment by maxammann** [comment]
Hi @Raj2032, I suppose you are referring to the stable implementation of MapLibre: maplibre-gl-js, maplibre-gl-native

maplibre-rs is an experimental project and far from being production ready.

The MapLibre project is not a Google Maps competition. MapLibre offers rendering engines for maps data and explicitly does not offer Software-as-a-Service products. Google Maps is a SaaS product.
It is non-comparable to MapLibre as such.

**Reply by Raj2032** [reply]
I see thanks mate
